app/assets/javascripts/admin/addon/templates/config-about.hbs

```diff
@@ -1,4 +1,4 @@
-<AdminPageHeader
+<DPageHeader
   @titleLabel="admin.config_areas.about.header"
   @descriptionLabelTranslated={{i18n
```

Comment:
@martin-brennan I think we shouldn't do this kind of stuff.  We should always consider the label we give here is translated. I get that we can think that passing a key is nice but it just creates a weird API, where you have support foo/fooTranslated for every similar properties.

I strongly recommend:

```gjs
@titleLabel={{i18n "admin.config_areas.about.header"}}
```

Reply:
Ok fair, I wasn't too happy with this double-up as well. Will change